.github/workflows/publish.yaml

```diff
@@ -25,6 +25,11 @@ jobs:
         uses: actions/checkout@v4
       - name: Import environment variables from file
         run: cat ".github/env" >> $GITHUB_ENV
+      - name: Reclaim disk space
```

Comment:
clever hehe

Reply:
I would prefer if the GH action ubuntu image was smaller :-/

Reply:
Can't we use maybe a custom image?

Reply:
Apparently not and the disk space seems to be a recurrent issue with GitHub runners.